docs/src/pages/[platform]/components/link/react.mdx

```diff
@@ -47,7 +47,7 @@ To create a Link which opens in a new tab, use the `isExternal` prop. Under the
 
 ### Routing Libraries
 
-To use the Link with React routing libraries, pass in routing components to the `as` prop. Below is an example using React Router:
+You can use a Link with any React routing libraries that support custom component. Below is an example using Link with React Router v5, in which custom navigation component is allowed by passing it through the `component` prop:
```

Comment:
You can use a Link with any React routing library that supports custom components. Below is an example using Link with React Router v5, in which the Link is passed to the `component` prop as a custom navigation component.